src/Bridges/HttpDI/HttpExtension.php

```diff
@@ -26,6 +26,7 @@ class HttpExtension extends Nette\DI\CompilerExtension
 		'frames' => 'SAMEORIGIN', // X-Frame-Options
 		'csp' => [], // Content-Security-Policy
 		'csp-report' => [], // Content-Security-Policy-Report-Only
+		'fp' => [], // Feature-Policy
```

Comment:
`'feature-policy' => []` is IMHO better

Reply:
I thought so too, but I wanted to keep the same scheme as ´csp´.